extensions/parquet/table/src/brotliTest/java/io/deephaven/parquet/table/BrotliParquetTableReadWriteTest.java

```diff
@@ -94,24 +94,17 @@ public boolean equals(Object obj) {
         }
     }
 
-    private void compressionCodecTestHelper(final String codec) {
-        final String currentCodec = ParquetInstructions.getDefaultCompressionCodecName();
```

Comment:
The risk with removing this is that we might end up one day with a brotli codec that can be used cross-platform, and this might test method might run in the same jvm as other tests, contaminating them by changing state like this. Unless there is a good reason to remove this?

Reply:
I modified the test to keep it similar to how we expect users to specify compression codec. In this original implementation, we modify the default compression codec and then write the table with the default. This seems weird because users won't have the option to change the default.

Instead I tested it like we recommend here `https://deephaven.io/core/groovy/docs/how-to-guides/parquet-single/` by just passing the codec enum with the API call.

This should not contaminate any state in case this test is run on the same JVM as other tests.